css/donate.css

```diff
@@ -75,57 +81,53 @@ h2 {
   object-fit: contain; }
 
 .donate-impact {
-  background-color: #ece3f2;
+  background-color: rgba(236, 177, 206, 0.26);
   padding-top: 3%;
   padding-bottom: 3%; }
 
 .impact-header {
+  color: black;
   text-align: center;
-  padding-bottom: 1%; }
+  padding-bottom: 2.5%; }
 
-.donate-option1 {
-  background: #ffffff;
-  border: 6px solid #1cd5d5;
-  padding: 3%;
-  padding-top: 5%;
-  text-align: center;
-  width: 100%;
-  height: 100%; }
+.donate-option1-header, .donate-option2-header, .donate-option3-header {
+  display: inline-block;
+  font-size: 300%;
+  margin-bottom: 10%; }
 
 .donate-option1-header {
-  color: #1cd5d5;
-  font-size: 300%; }
-
-.donate-option2 {
-  background: #ffffff;
-  border: 6px solid #a88aa8;
-  padding: 3%;
-  padding-top: 5%;
-  text-align: center;
-  width: 100%;
-  height: 100%; }
+  color: #82ecec; }
 
 .donate-option2-header {
-  color: #a88aa8;
-  font-size: 300%; }
-
-.donate-option3 {
-  background: #ffffff;
-  border: 6px solid #ff5789;
-  padding: 3%;
-  padding-top: 5%;
-  text-align: center;
-  width: 100%;
-  height: 100%; }
+  color: #b5a1c4; }
 
 .donate-option3-header {
-  color: #ff5789;
-  font-size: 300%; }
+  color: #f27aaa; }
+
+.donate-option1, .donate-option2, .donate-option3 {
+  background: white;
+  border-radius: 33px;
+  padding: 11%;
+  margin-left: 10%;
+  margin-right: 10%;
+  width: 80%;
+  height: 100%;
+  min-height: 315px; }
```

Comment:
let's decrease slightly to 280px (to the point where height of all boxes match while being as short as possible)

Reply:
I think the min height for all to match is 315px if I leave padding as 11%. If you want the boxes to be more square-shaped (280px min height), I'm going to have to change padding to 7%. I think 8% padding and 290px min-height also works out pretty well.